composer.json

```diff
@@ -9,14 +9,14 @@
             "email": "jeremygiberson@gmail.com"
         }
     ],
-    "minimum-stability": "dev",
+    "minimum-stability": "stable",
     "require": {
         "php": ">=5.6",
         "symfony/console": "^3.0",
         "symfony/process": "^3.0",
         "composer/composer": "dev-master",
         "symfony/event-dispatcher": "^3.0",
-        "servo/fluidxml": "^1.13"
+        "servo/fluidxml": "1.13"
```

Comment:
when i updated to ^1.2 we got a truckload of test failures. i thought it best to lock that down temporarily.